### PR TITLE
Integration tests for acl:Append.

### PR DESCRIPTION
**Pull request by peichman-umd**
**JIRA Ticket**: https://jira.duraspace.org/browse/FCREPO-2845

# What does this Pull Request do?
Test append to container and append to binary.

# What's new?
Adds integration tests to confirm the functionality of acl:Append mode access for containers and for binaries.

# How should this be tested?
Run these tests! 😄 

# Additional Notes:
**Important:** The tests are *currently failing* because of issues identified in:
* https://jira.duraspace.org/browse/FCREPO-2857
* https://jira.duraspace.org/browse/FCREPO-2858

# Interested parties
@whikloj , @fcrepo4/committers

### CODE REVIEW

**Comment by whikloj** [comment]
This also covers https://jira.duraspace.org/browse/FCREPO-2857 now

**Comment by peichman-umd** [comment]
This also covers https://jira.duraspace.org/browse/FCREPO-2858 now

File: fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 1]
```diff
@@ -951,4 +966,108 @@ public void testOptionsWithReadOnlyUser() throws IOException {
         assertEquals(HttpStatus.SC_OK, getStatus(optionsReq));
     }
 
+    @Test
+    public void testAppendOnlyToContainer() throws IOException {
+        final String testObj = ingestObj("/rest/test_append");
+        final String acl = ingestAcl("fedoraAdmin", "/acls/23/acl.ttl", testObj + "/fcr:acl");
```
Comment (awoods) [inline fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: The `acl` variable is never used, and can be removed.

File: fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 2]
```diff
@@ -951,4 +966,108 @@ public void testOptionsWithReadOnlyUser() throws IOException {
...
+            logger.debug("Testing SPARQL update: {}", query);
+            assertEquals(HttpStatus.SC_NO_CONTENT, getStatus(patchReq));
+        }
+
+    }
+
+    @Test
+    public void testAppendOnlyToBinary() throws IOException {
+        final String testObj = ingestBinary("/rest/test_append_binary", new StringEntity("foo"));
+        final String acl = ingestAcl("fedoraAdmin", "/acls/24/acl.ttl", testObj + "/fcr:acl");
```
Comment (awoods) [inline fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: The `acl` variable is never used, and can be removed.

**Pull request merged**